### ISSUE TITLE
关于 CTCLoss

### ISSUE THREAD

**Issue by leonzhu211** [Bobo-y/CRNN_pytorch#1]
请问下：
你训练的过程中 loss 收敛吗？

官网的文档是
https://pytorch.org/docs/stable/nn.html?highlight=ctcloss#torch.nn.CTCLoss

loss = ctc_loss(input, target, input_lengths, target_lengths)

与实现中的 target_lenghts 和 input_lengths 反了

**Comment by Bobo-y** [comment]
@leonzhu211 我差点都😵了，我是以关键字传的参啊つ﹏⊂顺序想怎么写就怎么写啊

**Comment by leonzhu211** [comment]
@yl305237731 
是我错了~
